test/models/rss_log_test.rb

```diff
@@ -2,5 +2,44 @@
 
 require("test_helper")
 
+# tests of MO's "rss_logs"
 class RssLogTest < UnitTestCase
+  def test_url_for_normalized_controllers
+    normalized_rss_log_types.each do |type|
+      rss_log = rss_log_for_type(type)
+      id = rss_log.target_id
+      assert_match(type_normalized_show_path(type, id), rss_log.url,
+                   "rss_log.url incorrect for #{model(type)}")
+    end
+  end
+
+  # ---------- helpers ---------------------------------------------------------
+
+  def normalized_rss_log_types
+    RssLog.all_types.each_with_object([]) do |type, ary|
+      ary << type if model(type).controller_normalized?(model(type).name)
+    end
+  end
+
+  def model(type)
+    type.camelize.constantize
+  end
+
+  # rss_log factory
+  def rss_log_for_type(type)
```

Comment:
It would help me to emphasize that this is *creating* a sample rss_log object.  That would save me from having to parse this method to find out what it's doing.  Or a comment would work just as well, but those are frowned on I guess?  What about something like "create_test_rss_log" (since type is being passed as a parameter, the "for_type" is sorta implicit).

Reply:
Thank you!
Your name for that method is much better. I'll fix it.